tests/__init__.py

```diff
@@ -353,40 +353,80 @@ def dbus_con(self):
     def dbus_con_sys(self):
         return self.test_case.dbus_con_sys
 
-    def start_impl_portal(self, params=None, portal=None):
-        """
-        Start the impl.portal for the given portal name. If missing,
-        the portal name is derived from the class name of the test, e.g.
-        ``TestFoo`` will start ``org.freedesktop.impl.portal.Foo``.
-        """
-        portal = portal or self.portal_name
-        self.p_mock, self.obj_portal = self.test_case.spawn_server_template(
-            template=f"tests/templates/{portal.lower()}.py",
-            parameters=params,
+    def get_bus_for_template(self, template: str):
+        # we need the bus address from the template module
+        module = dbusmock.mockobject.load_module(template)
+
+        bustype = dbusmock.BusType.SYSTEM if module.SYSTEM_BUS else dbusmock.BusType.SESSION
+        assert bustype is not None
```

Comment:
can't ever happen - is this to satisfy some naive linter?

Reply:
Copied from python-dbusmock but you're right. Removed it.